sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -1655,7 +1655,7 @@ var setParentClass = function(child, parent) {
         var fract = x - truncated;
```

Comment:
The JS implementation is exactly the same with the native code, see the function applyPoint() in native deqp: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#3168. The the result of this line differs from that in native code. When the input x equals to inf / -inf, the result is NaN. Then neither of these two ifs below (between line 1658 and line 1661) will be executed.